zadaci/sa-kolokvijuma/2020/RA/T5/G6/zadatak.rst

```diff
@@ -0,0 +1,28 @@
+Iz zadate datoteke učitati podatke u jednostruko spregnutu listu (podaci se dodaju u sortiranom redosledu na osnovu naziva filma), pri čemu se struktura ``film_st`` sastoji od sledećih polja:
+
+- naziv filma (jedna reč, do 30 karaktera)
+- cena ulaznice (realan broj)
+- broj ulaznica (pozitivna celobrojna vrednost)
+
+
+Naravno, struktura ``film_st`` sadrži i podatke potrebne za pravilno formiranje jednostruko spregnute liste. Potrebno je izračunati ukupnu zaradu po filmu, kao i ukupnu zaradu za sve filmove. Rezultat je potrebno ispisati na standardni izlaz. Naziv ulazne datoteke se prosleđuje kroz parametre komandne linije.
+
+Primer poziva programa::
+
+    ./a.out filmovi.txt
+
+sa zadatim ulazom u datoteci ``filmovi.txt``::
+
+    Pokajanje 250 8
+    Hari_Poter_2 300 10
+    Gospodar_prstenova_1 400 6
+    Hari_Poter_4 400 30
```

Comment:
Adding input file is not necessary if you're going to copy its contents into zadatak.rst. I'd suggest that you lose one colon in line 14 and change lines 16-19 with:

.. literalinclude:: filmovi.txt

Reason why I prefer doing like this is that there's almost no chance that program input will differ with input provided in the assignment text.